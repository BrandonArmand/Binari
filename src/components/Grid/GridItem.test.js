import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import GridItem from "./GridItem";

describe("GridItem", () => {
    it("renders snapshot", () => {
        const { container } = render(<GridItem />);
        expect(container).toMatchSnapshot();
    });
    it("renders GridItem", () => {
        render(<GridItem>GridItem</GridItem>);
        expect(screen.getByText("GridItem")).toBeInTheDocument();
    });
});
