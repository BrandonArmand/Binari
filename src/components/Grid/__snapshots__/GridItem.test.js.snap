// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridItem renders snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root makeStyles-grid-1  MuiGrid-item"
  />
</div>
`;
